DQM/DTMonitorModule/python/dtDQMOfflineSources_Cosmics_cff.py

```diff
@@ -50,3 +50,12 @@
                                 dtEfficiencyMonitor +
                                 dtTriggerEfficiencyMonitor +
                                 dqmInfoDT)
+
+import EventFilter.DTRawToDigi.dturosunpacker_cfi
+_dturosunpacker = EventFilter.DTRawToDigi.dturosunpacker_cfi.dturosunpacker.clone()
+from Configuration.Eras.Modifier_run2_DT_2018_cff import run2_DT_2018
+run2_DT_2018.toReplaceWith(dtDataIntegrityUnpacker, _dturosunpacker)
+run2_DT_2018.toModify(DTDataIntegrityTask,FEDIDmin=cms.untracked.int32(1368))
```

Comment:
era based modifications must be included in the python file that defines the module (and not propagated into the many other configurations that import it)

Reply:
DTDataIntegrityTask is defined in DQM/DTMonitorModule/python/dtDataIntegrityTask_cfi.py